test/integration/component/test_cpu_max_limits.py

```diff
@@ -242,8 +242,9 @@ def test_02_deploy_vm_account_limit_reached(self):
         self.debug("Deploying instance with account: %s" %
                    self.child_do_admin.name)
 
-        self.createInstance(account=self.child_do_admin,
+        self.vm1 = self.createInstance(account=self.child_do_admin,
             service_off=self.service_offering, api_client=api_client_admin)
+        self.cleanup.append(self.service_offering)
```

Comment:
@Pearl1594 , why are you adding the offering to the cleanu[p list here?
It should be added right after creation and the tearDown method at line 75 should be fixed to use the parent implementation.

Reply:
Oh this was a mistake, my bad